guide/ops/configuration/brooklyn_cfg.md

```diff
@@ -185,9 +185,23 @@ or
 [EntitlementManager]({{book.url.brooklyn_javadoc}}/org/apache/brooklyn/api/mgmt/entitlement/EntitlementManager.html).
 
 
-
 ## HTTPS Configuration
 
 See [HTTPS Configuration]({{book.path.docs}}/ops/configuration/https.md) for general information on configuring HTTPS.
 
 
+## Session configuration
+
+Apache Brooklyn uses a util class, `org.apache.brooklyn.rest.util.MultiSessionAttributeAdapter` for ensuring requests 
+in different bundles can get a consistent shared view of the data stored in the session.
+
+This class marks as used the session on the other modules by resetting the max inactive interval and also allows to 
```

Comment:
Could you add a bit of detail here, this is quite terse - maybe describe the flow a little?  "When a request on a given session is received...."